xDrip/Scenes/Home/Home/HomeRouter.swift

```diff
@@ -25,4 +25,10 @@ final class HomeRouter: NSObject, HomeRoutingLogic, HomeDataPassing {
     var dataStore: HomeDataStore?
     
     // MARK: Routing
+    
+    func routeToEntriesList() {
+        let controller = UIStoryboard(board: .entries).instantiateViewController(withIdentifier: "EntriesListNavigationController")
```

Comment:
Move "EntriesListNavigationController" to a static constant of `EntriesListRouter`.